### PR TITLE
10.4.0 upgrade

### PR DESCRIPTION
**Pull request by voroyam**
## 2020-03-18

* Changed
  * Upgrade ownCloud version from 10.3.2 to 10.4.0
  * Upgrade Collabora, OnlyOffice
  * Added behat

Removed the env variable section in the readme file, since it was empty.

### CODE REVIEW

**Comment by micbar** [comment]
@voroyam Can you remove the last commit "behat fix"

File: .drone.star [hunk 1]
```diff
@@ -2,20 +2,35 @@ def main(ctx):
...
-      'richdocuments': 'https://marketplace.owncloud.com/api/v1/apps/richdocuments/2.2.0',
-      'richdocuments_sha': 'fb0d7668fd1e2a5e18d569ee3849ff0029476f3fba3eac181c326a9a7d99be1b',
-      'onlyoffice': 'https://marketplace.owncloud.com/api/v1/apps/onlyoffice/4.0.1',
-      'onlyoffice_sha': 'b4bef78d11938d3892c4c5993cab110c149e0aadef0e60639513d9c1c2568cb1',
+      'richdocuments': 'https://marketplace.owncloud.com/api/v1/apps/richdocuments/2.3.0',
+      'richdocuments_sha': '5fe7bbe11f9f3c9d7d7eddc9a029e5cbb643834e0105cd89f377c36fdd033ecb',
+      'onlyoffice': 'https://marketplace.owncloud.com/api/v1/apps/onlyoffice/4.1.3',
+      'onlyoffice_sha': '8a4b626d4a522d81be6443bbfec42a6895f20d0ae61a2e5fea26ee96497e635e',
       'php': '7.3',
       'behat_version': '10.3.2',
```
Comment (tboerger) [inline .drone.star]: Set this to `10.4.0`

File: README.md [hunk 2]
```diff
@@ -24,12 +24,6 @@ For a guide how to get started with this Docker image please take a look at our
 
 * 8080
 
-## Available environment variables
```
Comment (tboerger) [inline README.md]: This is intentionally added to make clear this docker image does not define specific env variables, it's structured like that for all docker image repos.

**Comment by micbar** [comment]
@voroyam Let us see, CI should now pass.

**Comment by voroyam** [comment]
Nice!

**Pull request merged**